caasp-istio-proxyv2-image/_service

```diff
@@ -0,0 +1,10 @@
+<services>
+    <service name="replace_using_package_version" mode="buildtime">
+        <param name="file">caasp-istio-proxyv2-image.kiwi</param>
+        <param name="regex">1.5.4</param>
```

Comment:
Note that doing so will replace any `1.5.4` reference to the version of the `cilium-istio-proxy` package... I doubt this is intented

Reply:
the image will now be reverted to use %%PKG_VERSION%% and the cilium-istio-proxy package is now being modified to build as version 1.5.4. So this hard-coded tagging can be removed and _service file can be retained.